python-regression/tests/features/machine2/2_transaction_tests.feature

```diff
@@ -30,18 +38,25 @@ Feature: Test transaction confirmation
 
 
         When a transaction is generated and attached on "nodeA-m2" with:
-            | keys    | values       | type        |
-            | address | TEST_ADDRESS | staticValue |
-            | value   | 0            | int         |
+        | keys                  | values                    | type          |
+        | address               | TEST_ADDRESS              | staticValue   |
+        | value                 | 0                         | int           |
 
         And "getInclusionStates" is called on "nodeA-m2" with:
-            | keys         | values             | type        |
-            | transactions | TEST_STORE_ADDRESS | staticList  |
-            | tips         | latestMilestone    | configValue |
+        | keys                  | values                    | type          |
+        | transactions          | TEST_STORE_ADDRESS        | staticList    |
+        | tips                  | latestMilestone           | configValue   |
 
         Then the response for "getInclusionStates" should return with:
-            | keys   | values | type          |
-            | states | False  | boolListMixed |
+        | keys                  | values                    | type          |
+        | states                | False                     | boolListMixed |
+
+        And the zmq stream for "nodeA-m2" contains a response for following topics:
+        |keys                   |
+        |sn                     |
+        |sn_trytes              |
+        |tx                     |
+        |tx_trytes              |
```

Comment:
Not particularly, these transactions are just created and sent out so the transaction hashes and trytes are not hardcoded anywhere. I'd have to move the test to a different machine that uses a hardcoded transaction, which can be problematic because the sn and sn_trytes require the transaction to be solidified, which requires a milestone to be issued. So either I write a new scenario just for this part, or I rewrite one of the other tests to accommodate that.